src/DevLicenseMeta.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: AGPL-3.0
```

Comment:
We must also be consistent regarding licenses. In the other contracts we always use //SPDX-License-Identifier: BUSL-1.1, but I would check with Rob just to be sure.